net-misc/ntpsec/ntpsec-9999.ebuild

```diff
@@ -164,12 +153,18 @@ src_install() {
 
 	# move doc files to /usr/share/doc/"${P}"
 	use doc && mv -v "${ED}"/usr/share/doc/"${PN}" "${ED}"/usr/share/doc/"${P}"/html
-}
-
-python_install() {
+	local impl
+	for impl in "${_PYTHON_SUPPORTED_IMPLS[@]}"; do
+		local instpath=${d#${D%/}}
```

Comment:
You mentioned you're not going to change it, but I'm not clear on why. In EAPI 7+, it's [guaranteed](https://mgorny.pl/articles/the-ultimate-guide-to-eapi-7.html#d-ed-root-eroot-no-longer-have-a-trailing-slash) to not have a trailing slash.

Reply:
It's also unused. I am just too goofy right now to read it. I have commit dropping them and I'll push it tomorrow probably.